### DISCUSSION TITLE
I just want to say thank you!

### DISCUSSION THREAD

**Discussion by jakejscott** [AllocZero/EfficientDynamoDb#199] [General]
This library is so awesome. We've been using on a clients project and it's by far the best DynamoDB library I've come across. 

If there's any way I can help out with it let me know.

Thanks again
Jake

**Comment by firenero** [comment]
Hi @jakejscott. 
I'm glad to hear that people use our library in real-world projects.

It would be great to see a detailed feedback about your experience with the library. What issues did you encounter, what do you think we can improve, etc.
